components/src/core/components/Calendar/Calendar.vue

```diff
@@ -203,6 +203,12 @@ export default defineComponent({
               offset: i === 0 ? getDayOffset(date, this.firstDayOfWeek) : 0,
               attributes: this.attributes[i],
               event: this.parsedEvents[i],
+              onKeyup: ($event: KeyboardEvent) => {
```

Comment:
use event modifiers. both for enter key and stop propagation

Reply:
Is it possible to use @keyup.enter.stop="" here? Since its a render function. ? its giving me an error like it cannot be used here, please can you help?

Reply:
my mistake, didn't notice it's a render function.